summerofcode/2023.md

```diff
@@ -129,6 +136,20 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Difficulty: Hard
 - Upstream Issues (URL): https://github.com/kubevela/kubevela/issues/5366
 
+#### KubeVela HA Enhancement (VelaHA)
+
+- Description: KubeVela can be deployed in a single management cluster.The KubeVela management cluster has a single point of risk. When KubeVela management clusterdisaster occurs, business continuity cannot be guaranteed. In the production environment of enterprise use case(financial industry), KubeVela needs to have higher availability requirements.
+  Consider three enhancements to achieve this goal.
+* Cluster Meta backup & recovery,
+* Master redundancy,
+* Failover support
+- Expected outcome: velaha as addon for KubeVela with command line. cli> vela ha enable
+- Recommended Skills: Golang, Kubernetes, KubeVela, CUELang
+- Mentor(s): Jiahang Xu @jefree-cat jiahang_xu@cmbchina.com, Xiangbo Ma @fourierr maxiangboo@gmail.com
```

Comment:
👍 , mentor(s) are in https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L876